libmscore/chordrest.cpp

```diff
@@ -1109,6 +1109,16 @@ bool ChordRest::setProperty(P_ID propertyId, const QVariant& v)
       return true;
       }
 
+void ChordRest::supportedProperties(QList<P_ID>& dest, bool writeable)
+      {
+      DurationElement::supportedProperties(dest, writeable);
+      if (writeable) {
+            dest << P_ID::SMALL << P_ID::BEAM_MODE << P_ID::STAFF_MOVE << P_ID::DURATION_TYPE << P_ID::VISIBLE;
+            } else {
+            dest << P_ID::SMALL << P_ID::BEAM_MODE << P_ID::STAFF_MOVE << P_ID::DURATION_TYPE;
+            }
```

Comment:
IMHO better with less braces here, also further up in other single line stamtents following an `if` or `for`.
Makes it easier to read, I think. 
Here not even an `else` seems needed, if the statement following were moved to before the `if`?
Matter if personal taste, mostly...